receiver/haproxyreceiver/metadata.yaml

```diff
@@ -29,6 +23,14 @@ resource_attributes:
     description: load balancing algorithm
     type: string
 
+attributes:
+  proxy_name:
+    description: Proxy name
+    type: string
+  service_name:
+    description: Service name (FRONTEND for frontend, BACKEND for backend, any name for server/listener)
+    type: string
```

Comment:
Should either or both of these be resource attributes?

Reply:
They might - we're still working out the metrics. This PR defines just one metric to test out the socket read and CSV export.